src/annotator/guest.js

```diff
@@ -131,6 +131,7 @@ export default class Guest {
     this._hostFrame = hostFrame;
     this._highlightsVisible = false;
     this._isAdderVisible = false;
+    this.selectedRanges = /** @type {Range[]} */ []; // ranges of the current text selection
```

Comment:
```suggestion
    /** Ranges of the current text selection. */
    this.selectedRanges = /** @type {Range[]} */ ([]);
```

- TS requires the `@type`-d expression to be wrapped in parens
- In order for the symbol documentation to be attached to the symbol (eg. for use in VS Code tooltips and similar in other IDEs), you need to use JSDoc comments.